component-samples/demo/rv/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 # Copyright 2020 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
-version: "2.4"
+version: "3.4"
```

Comment:
Should be 2.4 right ?

Reply:
fixed in commit ac5c71922b41df4598d1eb9b068ce0a5edfe89fc